client/my-sites/plugins/minimum-jetpack-version-notice.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { warningNotice } from 'state/notices/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	isJetpackSite,
+	siteHasMinimumJetpackVersion,
+	getSiteAdminUrl,
+	getSiteDomain
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends Component {
+
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: PropTypes.bool.isRequired,
+		adminUrl: PropTypes.string,
+		domain: PropTypes.string,
+	};
+
+	showWarning( { minimumJetpackVersionFailed, domain, adminUrl, triggerNotice, translate, siteId } ) {
```

Comment:
I understand that, @jorgefilipecosta. What I mean is that in the ideal case we'd have notices being shown as a result of a redux action being dispatched. This allows for a more centralized and easy to test notice system. You can see such examples in state/notices/middleware.js.

However, I also realize that's doing it with the middleware is not easy to achieve with the current plugins implementation, so I'm good with what you're suggesting, as long as we add a TODO comment that this should later be migrated to use the notices middleware.